packages/logs/src/boot/startLogs.ts

```diff
@@ -63,30 +67,47 @@ export function doStartLogs(
   }
 
   function reportError(error: RawError) {
-    errorLogger.error(
-      error.message,
-      combine(
-        {
-          date: error.startClocks.timeStamp,
-          error: {
-            kind: error.type,
-            origin: error.source,
-            stack: error.stack,
-          },
+    const messageContext: Partial<LogsEvent> = {
+      date: error.startClocks.timeStamp,
+      error: {
+        kind: error.type,
+        origin: error.source,
+        stack: error.stack,
+      },
+    }
+    if (error.resource) {
+      messageContext.http = {
+        method: error.resource.method as any, // TODO: Set method to upper case (it's a breaking change),
+        status_code: error.resource.statusCode,
+        url: error.resource.url,
+      }
+    }
+    logger.error(error.message, messageContext)
+  }
+
+  const LogStatusForApi = {
+    [ConsoleApiName.log]: StatusType.info,
+    [ConsoleApiName.debug]: StatusType.debug,
+    [ConsoleApiName.info]: StatusType.info,
+    [ConsoleApiName.warn]: StatusType.warn,
+    [ConsoleApiName.error]: StatusType.error,
+  }
+
+  function reportConsoleLog(log: ConsoleLog) {
+    let messageContext: Partial<LogsEvent> | undefined
+    if (log.apiName === ConsoleApiName.error) {
+      messageContext = {
+        error: {
+          origin: ErrorSource.CONSOLE,
+          stack: log.stack,
         },
-        error.resource
-          ? {
-              http: {
-                method: error.resource.method,
-                status_code: error.resource.statusCode,
-                url: error.resource.url,
-              },
-            }
-          : undefined
-      )
-    )
+      }
+    }
+    logger.log(log.message, messageContext, LogStatusForApi[log.apiName])
   }
+
   errorObservable.subscribe(reportError)
+  consoleObservable.subscribe(reportConsoleLog)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏it may just be me but I was a bit bugged by the fact that console errors don't go through `reportError` since they are ... errors 🤔.
Maybe tweaking the namings to`rawErrorObservable` and `reportRawError` could help.
wdyt?

Reply:
I see. I'll update the names. I keep this comment in mind for log assembly improvement task.